### ISSUE TITLE
More detailed ReadMe instruction

### ISSUE THREAD

**Issue by shawnkoon** [darcikhey27/POST-Java-Reset#1]
This seems like it is an awesome project.
I would like to try it out but need to know if there is pre-required setup and how to run it.

**Comment by darcikhey27** [comment]
yes, it will be awesome. I will add a detailed ReadMe file and I will write down
pre-required setup instructions

**Comment by darcikhey27** [comment]
Almost done with the project. I got POST requests being sent to nodes resetting and deleting stuff from them. I will update later, I'm working on polishing up the code with better data structures and better esthetics for the GUI. @shawnkoon
